tests/components/fan/common.py

```diff
@@ -154,22 +154,12 @@ async def async_set_direction(
 class MockFan(MockEntity, FanEntity):
     """Mock Fan class."""
 
-    @property
-    def preset_mode(self) -> str | None:
-        """Return preset mode."""
-        return self._handle("preset_mode")
-
     @property
     def preset_modes(self) -> list[str] | None:
         """Return preset mode."""
         return self._handle("preset_modes")
 
-    @property
-    def supported_features(self):
-        """Return the class of this fan."""
-        return self._handle("supported_features")
-
-    async def async_set_preset_mode(self, preset_mode: str) -> None:
+    def set_preset_mode(self, preset_mode: str) -> None:
         """Set preset mode."""
         self._attr_preset_mode = preset_mode
         self.async_write_ha_state()
```

Comment:
We can remove this since the state will be updated after the service call since it's a polling entity.
```suggestion
```